Makefile

```diff
@@ -36,6 +38,14 @@ build: layout
 build-binary:
 	go build -o PolicyGenerator cmd/main.go
 
+build-so:
+	@mkdir -p build_output
+		go build -buildmode=c-shared -o build_output/_processGeneratorConfigC_$(OS)_$(ARCH).so cmd/main.go
+
+build-so-multiarch:
+	@mkdir -p build_output
+	GOOS=linux CGO_ENABLED=1 GOARCH=amd64 go build -o build_output/_processGeneratorConfigC_linux_amd64.so cmd/main.go
+
```

Comment:
This Make target throws all sorts of errors on my MacBook, but is this Linux-specific? The "multiarch" name is throwing me off a bit.

Reply:
yah i forgot to add other archs....also cross compile CGO on mac... is hard

i can remove this target for now